### ISSUE TITLE
Split list into lettered sections

### ISSUE THREAD

**Issue by svillegascreative** [svillegascreative/vue-bingo#8]
## Problem
The list of options (on small screens) is long and hard to relate to the grid. It should be broken down into smaller sections representing the options in each column (B-I-N-G-O)
